controllers/courseController.js

```diff
@@ -1,28 +1,27 @@
 const fs = require('fs');
-const {getCourses,getCourseById,modifiedCoursesArray,modifiedCoursesArrayAfterUpdation,modifiedCoursesArrayAfterDeletion} = require('./../model/course');
-//let courses = JSON.parse(fs.readFileSync(`${__dirname}/../data/courses.json`));
+const {getCourses,getCourse,addCourse,updatedCourses,updatedCoursesAfterDeletion} = require('./../model/course');
 
 const getAllCourses = (req,res)=>{
     const courses = getCourses();
     res.status(200).json(courses);
 };
 
-const getCourse = (req,res)=>{
+const getACourse = (req,res)=>{
     const subjectId = req.params.subjectId * 1;
-    const course = getCourseById(subjectId);
+    const course = getCourse(subjectId);
     (course == null)? res.status(404).json({message : 'Course not found'}) : res.status(200).json(course);
 };
 
-const addCourse = (req,res)=>{
-    const courses = modifiedCoursesArray(req.body);
+const addNewCourse = (req,res)=>{
+    const courses = addCourse(req.body);
     fs.writeFile(`C:/express-assignment/data/courses.json`,JSON.stringify(courses),(err)=>{
```

Comment:
write operation should be in the model not in controller

Reply:
okay ok